### PR TITLE
[SW-534, SW-821] Improve wording in the transaction list

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/popup/components/ConsensusLabel.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default defineComponent({
     confirmationsRequired: { type: Number, required: true },
     defaultConfirmedBy: { type: Number, default: 0 },
     hasPendingTransaction: Boolean,
+    hasDot: Boolean,
```
Comment (peronczyk) [inline src/popup/components/ConsensusLabel.vue:19]: This looks rather funny :) I suggest to align with Tsvetan if this component should end with dot or not.

File: src/popup/components/ConsensusReachedLabel.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    hasPendingTransaction: Boolean,
+  },
+  setup(props) {
+    const localConfirmedBy = computed(
+      () => (props.hasPendingTransaction)
+        ? props.confirmedBy?.length
+        : props.defaultConfirmedBy,
+    );
+
+    const isConsensusReached = computed(
```
Comment (peronczyk) [inline src/popup/components/ConsensusReachedLabel.vue:48]: The component name is `Consesus Reached...` so by reading it I would assume that I should display it when the consensus is ready. But the logic encapsulated here states something opposite. So in my opinion the general naming convention is misleading here.

**Review by peronczyk — CHANGES_REQUESTED** [review]

**Review by konradodo — APPROVED** [review]

File: src/popup/components/ConsensusApprovedLabel.vue [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      {{ $t('multisig.consensusPending') }}
+    </template>
+      &hairsp;
+    <ConsensusLabel
+      :confirmations-required="confirmationsRequired"
+      :has-pending-transaction="hasPendingTransaction"
+      :default-confirmed-by="defaultConfirmedBy"
+      :confirmed-by="confirmedBy"
+      :signers="signers"
+      :has-dot="isConsensusReached"
```
Comment (peronczyk) [inline src/popup/components/ConsensusApprovedLabel.vue:17]: I guess this can be removed.

File: src/popup/components/ConsensusApprovedLabel.vue [hunk 4]
```diff
@@ -0,0 +1,77 @@
+<template>
+  <span class="consensus-approved-label">
+    <template v-if="isConsensusReached">
+      {{ $t('multisig.approved') }}
+    </template>
+    <template v-else>
+      <PendingIcon class="icon" />
+      {{ $t('multisig.consensusPending') }}
+    </template>
+      &hairsp;
```
Comment (peronczyk) [inline src/popup/components/ConsensusApprovedLabel.vue:10]: Wrong indentation.

**Pull request merged**